assets/training/distillation/components/data_generation/src/generate_data.py

```diff
@@ -539,10 +554,13 @@ def data_import(args: Namespace):
         raise Exception(
             f"Invalid request_batch_size. Value should be 0<=val<={MAX_BATCH_SIZE}, but it is {request_batch_size}")
 
+    if enable_cod and data_generation_task_type != DataGenerationTaskType.SUMMARIZATION:
```

Comment:
@sharvin2187 Can you please sync up with @ShobithNandakumar to make sure that the input data validation is also added for the summarization task as well.